README.md

```diff
@@ -0,0 +1 @@
+# have-i-fed-the-cat-app
```

Comment:
Always god to include a bit more info in here, around things like setup, what tech is used, what routes are available etc.